parser-typechecker/src/Unison/Hashing/V2/Referent.hs

```diff
@@ -27,7 +28,7 @@ type ConstructorId = Int
 pattern Ref :: Reference -> Referent
 pattern Ref r = Ref' r
 pattern Con :: Reference -> ConstructorId -> ConstructorType -> Referent
-pattern Con r i t = Con' r i t
+pattern Con r i t = Con' (ConstructorReference r i) t
```

Comment:
Yep, totally. Hashing should not pull in `Unison.ConstructorReference`, but I figured in the meantime it's ok to not define a `Hashing.ConstructorReference` right here and now because hashing already tugs on `Unison.Referent'`.